src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -77,20 +77,29 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
       this.stopPolling();
 
-      // Wait 1 min before polling again & do not show error message when encounter rate limit error
-      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests') ||
-        error.responseJSON?.errorCode === 'E0000047') {
+      // Polling needs to be stopped if it's a form error and session is invalid
+      if (error.responseJSON?.errorSummaryKeys?.includes('idx.session.expired')) {
+        return;
```

Comment:
why empty `return` ?

Reply:
empty return to exit the function since the rest of the checks isn't required for `idx.session.expired` error

Reply:
ahh then change the comment above.. Because we are not stopping anything here.. polling is already stopped.

Reply:
yap...i add comment closer to here so it's more self-explanatory